flink-shaded-hadoop/pom.xml

```diff
@@ -111,6 +111,7 @@ under the License.
 									<include>io.netty:netty:*</include>
 									<include>org.apache.curator:*</include>
 									<include>org.apache.hadoop:*</include>
+									<include>org.mortbay.jetty:*</include>
```

Comment:
I think it is not necessary to include the dependency in the fat jar. We only include those where the classes should be relocated to "hide" the dependency. We probably do not want to "hide" jetty.